konnect-runtime-setup.sh

```diff
@@ -122,6 +122,12 @@ check_variables() {
         error "Konnect runtime image name is missing"
     fi
 
+        # temporary fix for multiplatform support
+    if [[ $KONNECT_RUNTIME_IMAGE = "kong-gateway:3.0.0.0" ]]; then
+        echo "setting alpine tag"
+        KONNECT_RUNTIME_IMAGE="kong-gateway:3.0.0.0-alpine"
+    fi
+
```

Comment:
WDYT about having an array that contains all the Kong Gateway versions that are going to require this "hack"? Until the Ubuntu image is updated to handle multi-platform support all we need to do is add a new entry to the array and the conditional will still apply.

```suggestion
    # temporary fix for multi-platform support
    VERSIONS_TO_APPLY_ALPINE_SUFFIX=(
      "kong/kong-gateway:3.0.0.0"
    )
    if [[ "${VERSIONS_TO_APPLY_ALPINE_SUFFIX[*]}" =~ ${KONNECT_RUNTIME_IMAGE} ]]; then
      echo "setting alpine tag"
      KONNECT_RUNTIME_IMAGE="$KONNECT_RUNTIME_IMAGE-alpine"
    fi

```

Reply:
Added. As we discussed elsewhere, there are a couple of ways to address applying the change to versions as updates roll out. It's not very elegant to have to update a list of versions each time, but we can expect this feature to be short-lived and should avoid scripting advanced version detection. 

If we need a longer-term solution, I recommend we fix this problem elsewhere (ideally the existing API call that returns the config with the image version).

Reply:
Agreed, this is good enough for now. I hope at a minimum this will be only for 3.0.x.x releases; @hutchic do you have thoughts on when the now defaulting Ubuntu images will support multi-platform?